### ISSUE TITLE
Create new release for Fedora 39

### ISSUE THREAD

**Issue by septatrix** [philn/cog-fedora#1]
Hi, while trying to update our Fedora based image I noticed that there is no cog package for Fedora 39. Could you please add a build for Fedora 39 (and maybe even update cog to the 0.18.2 release)?

**Comment by septatrix** [comment]
PS: Have you tried or though about getting this included in the official Fedora repos? Debian also has cog, and so does Ubuntu (though weirdly only for LTS releases)

**Comment by philn** [comment]
I can update the package to 0.18.2, but not 0.19. Once 0.20 is out i can update to that stable release.

I don't plan to submit this to the official repo, sorry :) Too many things on my plate.

**Comment by septatrix** [comment]
> I can update the package to 0.18.2, but not 0.19. Once 0.20 is out i can update to that stable release.

Thanks. Having it in f39 is sufficient for me right now.

> I don't plan to submit this to the official repo, sorry :) Too many things on my plate.

Would you mind if I took your spec files and tried to upstream them? You currently have no license listed.

**Comment by philn** [comment]
Sure! Would you take care of wpewebkit too?

**Comment by septatrix** [comment]
Yes I could to that. I am currently familiarizing myself with the process of claiming ownership of an abandoned package in Fedora
